packages/video_player/video_player/lib/video_player.dart

```diff
@@ -17,7 +17,10 @@ export 'package:video_player_platform_interface/video_player_platform_interface.
 import 'src/closed_caption_file.dart';
 export 'src/closed_caption_file.dart';
 
-final VideoPlayerPlatform _videoPlayerPlatform = VideoPlayerPlatform.instance
+/// This is just exposed for testing. It shouldn't be used by anyone depending
+/// on the plugin.
+@visibleForTesting
+final VideoPlayerPlatform videoPlayerPlatform = VideoPlayerPlatform.instance
```

Comment:
Instead of making this visibleForTesting, can we just access `VideoPlayerPlatform.instance` on the tests? It's already public anyway!

Reply:
Removed this change as no longer required